### PR TITLE
LPS-142722 DM shortcuts item selector video isn't working

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
@boton 

The other approach we talk about. With the "type" att.

### CODE REVIEW

**Review by boton — CHANGES_REQUESTED** [review]
LGTM though I missing the deletion of file [constants.js](https://github.com/liferay-lima/liferay-portal/blob/9945eb4fa0b65adf18f55606228876ea69845445/modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/constants.js) because isn't used anymore.

File: modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/FileEntryFileEntryItemSelectorReturnTypeResolver.java [hunk 1]
```diff
@@ -79,12 +86,30 @@ public String getValue(FileEntry fileEntry, ThemeDisplay themeDisplay)
...
+				PropsValues.DL_FILE_ENTRY_PREVIEW_VIDEO_MIME_TYPES,
+				fileEntry.getMimeType()) ||
+			Objects.equals(
+				DLContentTypes.VIDEO_EXTERNAL_SHORTCUT,
+				fileEntry.getMimeType())) {
+
+			jsonObject.put(
+				"html",
+				_dlVideoRenderer.renderHTML(
+					fileEntry.getFileVersion(), themeDisplay.getRequest()));
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/FileEntryFileEntryItemSelectorReturnTypeResolver.java:101]: You're using the video renderer for both images and videos. Is that on purpose?

Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/FileEntryFileEntryItemSelectorReturnTypeResolver.java:101]: video render only if mimeType is DL_FILE_ENTRY_PREVIEW_VIDEO_MIME_TYPES or VIDEO_EXTERNAL_SHORTCUT

no images.

**Review by boton — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia Can you rebase and push-force? After that, do the forward yourself, everything looks good :+1:

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**